CommandsManual.md

```diff
@@ -274,36 +326,109 @@ Each Project file contains settings for a Project, an *impt* entity which links
 
 A Project file may affect commands called from the directory where the file is located. Product, Device Group, Devices, Deployment, and source code files referenced by Project file may be assumed by a command when they are not specified explicitly.
 
+**Note** Currently, Builder information in the project file must be created manually.
+
+Each project file contains the following:
+
+- Global Builder information: includes `variables` which are used by all device groups, and `libs` which are `.js` files that contain some functions used by builder variables
+- Device Groups: Each device group is a key-value pair where the key is the device gorup ID and the value is an object containing the following information:
+    - `endpoint`: an impCentral API endpoint (the API base URL).
+    - `accountID`: the account ID that the device group belongs to.
+    - `deviceFile`:The device source code file name
+    - `agentFile`: The agent source code file name
+    - Local Builder information: includes `variables` which are used the specific device group, and `preBuild` and `postBuild` which are commands or `.js` files that are executed before and after building the code.
+    - `isDefault`: Boolean flag to indicate whether this is the default device group to be used . It is only included in one of device groups.
```

Comment:
There's an extra space between "used" and the period at the end of the first sentence.

Also, what happens if "isDefault" is included more than once?

The 2nd sentence should probably be slightly reworded: "It must be included in one (and only one) of the device group entries."

Reply:
if it is included more than once, then the last account in the file with `isDefault` is going to be the default.